Formula/clang-format.rb

```diff
@@ -32,7 +32,6 @@ class ClangFormat < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "ninja" => :build
```

Comment:
Is there a reason why you dropped ninja? When building, ninja parallelizes by default. Make does not.

Reply:
We set `MAKEFLAGS` to parallelise `make` builds. The extra dependency shouldn't be needed.